docs/ReleaseHistory.md

```diff
@@ -11,6 +11,16 @@
 - FPS => False positive reduction in static analysis.
 - FNS => False negative reduction in static analysis.
 
+# UNRELEASED
+- DEP: `System.Buffers` dependency has been removed. The package no longer has any dependencies.
+- BRK: .NET 6.0 and 7.0 are no longer supported as they have reached end-of-life. Use a [supported version version of .NET](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core): .NET 8.0 or greater. 
+- BRK:  .NET Framework 4.5.1 through 4.6.0 are no longer supported. Use a version of .NET Framework version that supports [.NET Standard 2.0](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0): NET 4.6.1 or greater with .NET 4.7.2 or greater strongly recommended. Note that there are no supported versions of Windows that have a version of .NET Framework that would be affected by this change at runtime, but build changes may be required.
+- BRK: `IdentifiableScan` no longer supports stream input.
+- BUG: `SecretMasker` and `IdentifiableScan` considered non-alphanumeric delimiter preceding secret to be part of the match for `SEC101/200.CommonAnnotatedSecurityKey`.
+- BUG  `SEC101/061.LooseOAuth2BearerToken` had incorrect signatures, causing no matches to be found unless the input happened to also contain `sig=` or `ret=`
+- PRF: `IdentifiableScan` did not use high-performance scanning techniques for `SEC101/178.AzureIotHubIdentifiableKey` and `SEC101/200.CommonAnnotatedSecurityKey`. This was not intended, but a bug triggered fall-back to slower scanning due to incorrect signatures being used to determine which patterns could be matched with high performance.
```

Comment:
>to determine which patterns could be matched with high performance [](http://example.com/codeflow?start=275&length=66)

All this can be deleted, it follows naturally from the preceding description, IMO.

`to determine which patterns could be matched with high performance